vst.go

```diff
@@ -758,6 +763,27 @@ type (
 	}
 )
 
+// GetVal should be called in ProcessDoubleFunc or ProcessFloatFunc and will be called in GetDisplayVal.
+// It returns the plain Value or the return value of GetValue, when it was set for the Parameter
+func (e Parameter) GetVal() float32 {
+	if e.GetValue == nil {
+		return e.Value
+	} else {
+		return e.GetValue(e.Value)
+	}
+}
+
+// GetDisplayVal will be called in HostOpcode plugGetParamDisplay. Return a string formatted float value or the return
+// value of GetParamDisplay, when it was set for the Parameter
+func (e Parameter) GetDisplayVal() string {
+	if e.GetParamDisplay == nil {
+		e.ValueLabel = fmt.Sprintf("%f", e.GetVal())
+	} else {
+		e.ValueLabel = e.GetParamDisplay(e.GetVal())
+	}
+	return e.ValueLabel
```

Comment:
It seems that `ValueLabel` field can be removed now.

Reply:
Mhn, its needed in processor.go which is part of the VST Host part, right? I tried and changed it back.

Reply:
You are right. 

TBH, both `Parameters` and `Presets` fields can be safely removed from `Processor` struct. I assumed that they would be used for automation/pipe mutations, but it seems like having plugin would be enough.  

Since it'll simplify logic here, I propose to do it in this PR. Let me know what do you think.

Reply:
I followed your suggestion and removed it.